tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -31,25 +34,63 @@ class InactiveUser(Exception):
     pass
 
 
+class UnverifiedEmail(Exception):
+    """Unverified Email Error Handler."""
+
+    pass
+
+
+class ExpiredToken(Exception):
+    """Expired Token Error Handler."""
+
+    pass
+
+
 class TokenAuthorizationOIDC(ObtainAuthToken):
```

Comment:
This is now a parent class with abstract methods implemented by the respective AMS and LogivDotGov authorization classes below